includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1002,7 +1002,8 @@ public function generate_upe_checkout_experience_accepted_payments_html( $key, $
 
 		$data['description'] .= '</tbody>
 			</table>
-			<span id="wc_stripe_upe_change_notice" class="hidden">' . __( 'You must save your changes.', 'woocommerce-gateway-stripe' ) . '</span>';
+			<span id="wc_stripe_upe_change_notice" class="hidden">' . __( 'You must save your changes.', 'woocommerce-gateway-stripe' ) . '</span>
+			<p><a class="button" target="_blank" href="https://dashboard.stripe.com/account/payments/settings">' . __( 'Get more payment methods', 'woocommerce-gateway-stripe' ) . '</a></p>';
```

Comment:
I could create a new "field" and a new HTML generator (similar to `generate_upe_checkout_experience_accepted_payments_html`) for just the external link button. However, given that the external link is bind to the entire payment method table, I think the simplest way is to attach the button beneath the existing table's HTML. 

I added `<p>` to provide enough spacing.